packages/fxa-settings/src/components/Ready/index.tsx

```diff
@@ -22,40 +52,41 @@ const Ready = ({
   usePageViewEvent(viewName, {
     entrypoint_variation: 'react',
   });
+  const templateValues = getTemplateValues(viewName);
 
   return (
     <>
       <div className="mb-4">
-        <Localized id="ready-confirmation">
-          <h1 className="card-header">Your password has been reset</h1>
-        </Localized>
+        <h1 className="card-header">
+          <FtlMsg id={templateValues.headerId} attrs={{ header: true }}>
+            {templateValues.headerText}
+          </FtlMsg>
+        </h1>
       </div>
       <div className="flex justify-center mx-auto">
         <PulseHearts />
       </div>
       <section>
         <div className="error"></div>
-        <Localized id="ready-use-service">
-          <p className="my-4 text-sm">{`You’re now ready to use ${serviceName}`}</p>
-        </Localized>
+        <FtlMsg id="ready-use-service" vars={{ serviceName }}>
+          <p>You’re now ready to use {serviceName}</p>
```

Comment:
Hmm, ```<p className="my-4 text-sm">{`You’re now ready to use ${serviceName}`}</p>``` seems more correct to me than `<p>You’re now ready to use {serviceName}</p>`? From the variable name + class name on `p` perspective.

Reply:
My memory is that this caused errors, but I'll check it again -- maybe that was part of the `testAlll10n` issue?

Reply:
You were right! The tests are passing with that change also, I'll add it in!